jaas/src/main/java/com/griddynamics/jagger/jaas/service/DynamicDataService.java

```diff
@@ -66,11 +68,29 @@ public void destroy() {
         destroyerService.shutdown();
     }
     
-    protected void evictDataServiceFor(final Long configId) {
-        LOGGER.info("Evicting jagger test db config with id: {}", configId);
+    protected void evict(final Long configId) {
+        destroyerService.execute(() -> doEvict(configId));
+    }
+    
+    protected void evictIfAboveThreshold() {
```

Comment:
It seems quite strange to me. Maybe there is any possibility to make it in a simple way?